web-ui/src/components/feelings/Feelings.test.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import Feelings from "./Feelings";
+
+it("renders correctly", () => {
+  snapshot(<Feelings />);
+});
```

Comment:
Wouldn't hurt to have a test to make sure that the onSelect gets called with the appropriate value.